posts/indu-plp-part1/plp-part1.qmd

```diff
@@ -0,0 +1,217 @@
+
+You can find my [**PLP-Pipeline Project Link Here**](https://github.com/kosuri-indu/PLP-Pipeline)
+
+[**LinkedIn**](https://www.linkedin.com/in/kosuri-indu/) | [**GitHub**](https://github.com/kosuri-indu/)
+ 
+# Background
+
+## What is Observational Health?
+
+Observational health research involves studying patient data collected in real-world settings, typically through electronic health records (EHRs), claims databases, or registries. Unlike clinical trials, which are controlled and experimental, observational studies allow researchers to analyze the effects of treatments, interventions, and disease progressions in a more naturalistic setting. The advantage of observational health research is that it provides a vast, diverse dataset that reflects how treatments and conditions unfold in everyday clinical practice.
+
+However, analyzing observational health data can be challenging due to its complexity, unstructured nature, and biases inherent in real-world data. This is where standardized data models like OMOP CDM come into play.
+
+## What Is the OMOP CDM?
+
+The **Observational Medical Outcomes Partnership Common Data Model (OMOP CDM)** is a standardized framework for organizing and analyzing observational healthcare data. The OMOP CDM converts diverse sources of health data into a common format that supports large-scale, systematic analysis.
+
+The OMOP CDM organizes data into a consistent set of relational tables like `condition_occurrence`, `drug_exposure`, `person`, `visit_occurrence` etc, using standardized vocabularies. These tables are interconnected, allowing for relational analysis across a patient's medical history.
+
+By transforming diverse healthcare datasets into a common format, the OMOP CDM enables reproducibility, interoperability, and large-scale studies across institutions and populations.
+
+<br>
+<center>
+  ![](./omopcdm.png)
+
+  OMOP Common Data Model
+</center>
+
+## What is Patient-Level Prediction (PLP)?
+
+**Patient-Level Prediction (PLP)** refers to the use of machine learning or statistical models to estimate the likelihood of a specific clinical outcome for an individual patient, based on their personal medical history and characteristics.
+
+The key goal of PLP is to answer personalized clinical questions like:
+
+> *"For a patient who recently had a hospital visit for chest pain, what is their risk of experiencing a heart attack?"*
+
+PLP leverages patient data such as diagnoses, medications, procedures, and lifestyle factors and applies machine learning models to estimate the risk of future health events. Unlike traditional studies that provide population-level insights, PLP aims to provide targeted predictions that can help inform individual clinical decisions.
```

Comment:
I think you might want to contrast this with precision medicine. This almost sounds more like precision medicine than patient level prediction.